web/app/mu-plugins/disallow-indexing.php

```diff
@@ -3,12 +3,21 @@
 Plugin Name:  Disallow Indexing
 Plugin URI:   https://roots.io/bedrock/
 Description:  Disallow indexing of your site on non-production environments.
-Version:      1.0.0
+Version:      2.0.0
 Author:       Roots
 Author URI:   https://roots.io/
 License:      MIT License
```

Comment:
Do we want to add the text domain for "roots"? 💋

Reply:
this feature is becoming a beast